src/protocol/attribution/accumulate_credit.rs

```diff
@@ -22,6 +22,46 @@ impl AsRef<str> for Step {
     }
 }
 
+async fn accumulate_credit_cap_one<F, C, T>(
```

Comment:
would be nice to have some documentation for this function explaining the key differences